docs/sources/tutorials/get-started.md

```diff
@@ -196,60 +201,58 @@ This last component creates a [loki.write][] component named `grafana_loki` that
 This completes the simple configuration pipeline.
 
 {{< admonition type="tip" >}}
-The `basic_auth` is commented out because the local `docker compose` stack doesn't require it. 
-It is included in this example to show how you can configure authorization for other environments.
+The `basic_auth` is commented out because the local `docker compose` stack doesn't require it.
+It's included in this example to show how you can configure authorization for other environments.
 For further authorization options, refer to the [loki.write][] component reference.
 
 [loki.write]: ../../reference/components/loki.write/
 {{< /admonition >}}
 
-This connects directly to the Loki instance running in the Docker container.
+With this configuration, {{< param "PRODUCT_NAME" >}} connects directly to the Loki instance running in the Docker container.
 
 ## Reload the configuration
 
-Copy your local `config.alloy` file into the default configuration file location, which depends on your OS.
+1. Copy your local `config.alloy` file into the default configuration file location.
 
-{{< code >}}
+   {{< code >}}
 
-```macos
-sudo cp config.alloy $(brew --prefix)/etc/alloy/config.alloy
-```
+   ```macos
+   sudo cp config.alloy $(brew --prefix)/etc/alloy/config.alloy
+   ```
 
-```linux
-sudo cp config.alloy /etc/alloy/config.alloy
-```
+   ```linux
+   sudo cp config.alloy /etc/alloy/config.alloy
+   ```
 
-{{< /code >}}
+   {{< /code >}}
 
-Finally, call the reload endpoint to notify {{< param "PRODUCT_NAME" >}} to the configuration change without the need
-for restarting the system service.
+1. Call the `reload` endpoint to tell {{< param "PRODUCT_NAME" >}} to reload the configuration file without a system service restart.
```

Comment:
nit: my gut feeling that referring to the endpoint with backquotes turns it into a more technical reference. In that case, the technical endpoint is `/-/reload` instead of `reload`.